BankApplicationPractice/BankLibrary/Bank.cs

```diff
@@ -8,6 +8,8 @@ public class Bank<T> where T : IAccount
     {
         private readonly List<T> _accounts = new();
```

Comment:
remove this old collection and use only `AccountsCollection`. Add public methods identical to ones you use in `List<T>`